### ISSUE TITLE
module_init extra output (pid?) on every new terminal

### ISSUE THREAD

**Issue by Niek** [ohmyzsh/ohmyzsh#10014]
**Describe the bug**
After updating ohmyzsh, whenever I open a new terminal I get some output like this:
```shell
[3] 83148
```

**To Reproduce**
- Update ohmyzsdh
- Open a new terminal

**Expected behavior**
- Just the prompt should be shown

**Screenshots and/or Recordings**
<img width="305" alt="Screenshot 2021-06-30 at 14 42 05" src="https://user-images.githubusercontent.com/213140/123962069-585f2000-d9b1-11eb-9d6d-ae616af74f91.png">

**Desktop (please complete the following information):**

 - OS / Distro: macOS 11.4
 - Latest ohmyzsh update?: Yes
 - ZSH Version: zsh 5.8 (x86_64-apple-darwin20.0)
 - Terminal emulator: iTerm2

**Comment by Niek** [comment]
Apparently this was some kind of malware that added itself to my `~/.zsh`... Closing this now.

**Comment by rwmitchell** [comment]
Can you tell us more about the malware??  Do you know how you got it?  What does it do?  Given the output, it was running something in the background.

**Comment by Niek** [comment]
There was a `module_init` in the bottom of my `.zshrc`. Inside `.zsh/misc` there were 3 scripts: `module_init` (which ran a script ending with `& disown` which caused the PID to be printed) and a `login_helper` binary executable. I ran it through VT and got this: https://www.virustotal.com/gui/file/4c1edee6a399f11f99d17092aa138a22769f2a237b8d61368cc23b38061dc6b6/detection

I nuked all of the files so I don't have a backup now. No idea how I got this malware, but be careful...
